gdal/frmts/derived/deriveddataset.cpp

```diff
@@ -139,6 +139,19 @@ GDALDataset * DerivedDataset::Open( GDALOpenInfo * poOpenInfo )
     // Transfer metadata
     poDS->SetMetadata(poTmpDS->GetMetadata());
 
+    char ** papszMDList = poTmpDS->GetMetadataDomainList();
```

Comment:
this could be simplified as:
```
CSLConstList papszRPC = poTmpDS->GetMetadata("RPC");
if( papszRPC )
   poDS->SetMetadata(papszRPC, "RPC");
```

Reply:
Sorry, I get
```
C:\Users\yanns\git\gdal\gdal\frmts\derived\deriveddataset.cpp(145): error C2664: 'CPLErr VRTDataset::SetMetadata(char **,const char *)': cannot convert argument 1 from 'CSLConstList' to 'char **'
```

Reply:
ah right, forgot that SetMetadata() had a non const strict signature. Use char** instead of CSLConstList then